scripts/generate_astex_manifest.py

```
@@ -0,0 +1,159 @@
+            return 1
+        committed = OUT_CSV.read_text()
+        if committed != csv_text:
+            print(
+                f"FAIL: {OUT_CSV} drift vs canonical tree "
+                f"({n_files} files / {len(targets)} targets hashed)",
+                file=sys.stderr,
+            )
+            c_lines = committed.splitlines()
+            n_lines = csv_text.splitlines()
+            drift = 0
+            for a, b in zip(c_lines, n_lines):
```

Ruff (0.15.21)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)